### PR TITLE
Additional feedback

### PR DESCRIPTION
**Pull request by upalatucci**
Feedback doc: https://docs.google.com/document/d/1gVZP7ck3D7NhNMgjShyF1TqR5M1TQWvQNDR-x5KE9i0

- Interfaces types in horizontal rather than vertical (Network raw)
- Dark mode fixes
- feature flags in the drawer moved in the bottom
- fix popover link to the drawer
- vlan details in the drawer
- sort ports 

image
Screenshot from 2023-07-14 16-51-10

Screenshot from 2023-07-14 17-10-55
Screenshot from 2023-07-14 16-51-38

### CODE REVIEW

File: src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx [hunk 1]
```diff
@@ -12,35 +12,30 @@ type InterfaceDrawerDetailsTabProps = {
...
-                    <FlexItem>
-                      <Checkbox
-                        isDisabled
-                        id={`checkbox-${feature}`}
-                        value={feature}
-                        isChecked={selectedInterface.ethtool?.feature?.[feature]}
-                      />
-                    </FlexItem>
+      {selectedInterface.vlan && (
+        <div className="pf-u-mb-md" data-test="lldp-section">
```
Comment (hstastna) [inline src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx]: Do we really need this div? Maybe we could add `className` to `Flex` component below to style it, and this additional div wouldn't be needed. WDYT? Same for another div below in this file.

File: src/views/states/list/components/InterfacesTypeSection.tsx [hunk 2]
```diff
@@ -62,7 +63,7 @@ const InterfacesTypeSection: FC<InterfacesTypeSectionProps> = memo(
                   data-test={`${interfaceType}-${iface.name}-open-drawer`}
                 >
                   {iface.name}
-                  <Icon color="black" className="pf-u-mr-sm" />
+                  <Icon className="pf-u-mr-sm pf-u-color-200" />
```
Comment (hstastna) [inline src/views/states/list/components/InterfacesTypeSection.tsx]: I'd better start putting styling into separate css file, as it seems it is increasing here..

File: src/views/states/list/components/InterfacesTypeSection.tsx [hunk 3]
```diff
@@ -75,17 +76,17 @@ const InterfacesTypeSection: FC<InterfacesTypeSectionProps> = memo(
...
-                        {iface?.bridge?.port.map((portItem) => (
-                          <ListItem key={portItem.name}>{portItem.name}</ListItem>
+                        {ports.map((port) => (
+                          <ListItem key={port}>{port}</ListItem>
                         ))}
                       </List>
                     }
                   >
-                    <span> {iface?.bridge?.port?.length}</span>
+                    <span> {ports.length}</span>
```
Comment (hstastna) [inline src/views/states/list/components/InterfacesTypeSection.tsx]: Hmm, do we need that extra empty space here before `{ports.length}`?

File: src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx [hunk 4]
```diff
@@ -12,35 +12,30 @@ type InterfaceDrawerDetailsTabProps = {
...
+          <Title headingLevel="h4">VLAN details</Title>
```
Comment (hstastna) [inline src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx]: I think we are missing `t` function here for _VLAN details_ ;)

File: src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx [hunk 5]
```diff
@@ -12,35 +12,30 @@ type InterfaceDrawerDetailsTabProps = {
...
+            <FlexItem>
+              <strong>{t('Base interface')}:</strong>
+            </FlexItem>
+            <FlexItem>{selectedInterface.vlan['base-iface']}</FlexItem>
+          </Flex>
         </div>
       )}
+
       <div className="pf-u-mb-md" data-test="lldp-section">
         <Title headingLevel="h4">LLDP</Title>
```
Comment (hstastna) [inline src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx]: I can see here there's no t function used for LLDP. Don't we need it? I am not sure here...

**Review by hstastna — COMMENTED** [review]
Added a few comments, mostly nitpicks ;) 
Also I think we are missing translation in these places:

1. [Name](https://github.com/nmstate/nmstate-console-plugin/pull/65/files#diff-9589cf44c10fbdf765ee32ea03d06357cc6b6c7f4fd74c0f738a80b617dac7d9R54)
2. [IP address](https://github.com/nmstate/nmstate-console-plugin/pull/65/files#diff-9589cf44c10fbdf765ee32ea03d06357cc6b6c7f4fd74c0f738a80b617dac7d9R68)
3. [Ports](https://github.com/nmstate/nmstate-console-plugin/pull/65/files#diff-9589cf44c10fbdf765ee32ea03d06357cc6b6c7f4fd74c0f738a80b617dac7d9R82)

Aand maybe also here, I am not sure:

4. [LLDP](https://github.com/nmstate/nmstate-console-plugin/pull/65/files#diff-9589cf44c10fbdf765ee32ea03d06357cc6b6c7f4fd74c0f738a80b617dac7d9R88)
5. [MTU](https://github.com/nmstate/nmstate-console-plugin/pull/65/files#diff-9589cf44c10fbdf765ee32ea03d06357cc6b6c7f4fd74c0f738a80b617dac7d9L104)

PTAL. Thanks! :)

File: src/views/states/list/components/InterfacesTypeSection.tsx (hunk 3 above)
Comment (upalatucci) [reply src/views/states/list/components/InterfacesTypeSection.tsx]: No :-)

File: src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx (hunk 4 above)
Comment (upalatucci) [reply src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx]: Thanks

File: src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx (hunk 1 above)
Comment (upalatucci) [reply src/views/states/list/components/InterfaceDrawer/InterfaceDrawerDetailsTab.tsx]: Used `Stack` and `StackItem` with `hasGutter`

File: src/views/states/list/components/InterfacesTypeSection.tsx (hunk 2 above)
Comment (upalatucci) [reply src/views/states/list/components/InterfacesTypeSection.tsx]: Done

**Comment by upalatucci** [comment]
Thanks @hstastna resolved all the comments

**Review by hstastna — APPROVED** [review]

**Pull request merged**